packages/hub/services/serializers/push-notification-serializer.ts

```diff
@@ -0,0 +1,29 @@
+import { PushNotificationRegistration } from '../../routes/push_notification_registrations';
+
+interface JSONAPIDocument {
+  data: any;
+  included?: any[];
+}
+
+export default class PushNotificationRegistrationSerializer {
+  serialize(model: PushNotificationRegistration): JSONAPIDocument {
+    const result = {
+      data: {
+        type: 'push-notification-registration',
+        attributes: {
+          'owner-address': model.ownerAddress,
+          'push-client-id': model.pushClientId,
+          'disabled-at': model.disabledAt,
+        },
+      },
+    };
+
+    return result as JSONAPIDocument;
+  }
+}
+
+declare module '@cardstack/di' {
+  interface KnownServices {
+    'push-notification-registration-serializer': PushNotificationRegistrationSerializer;
```

Comment:
Given that this has `registration` in it, should the filename have that as well?